modules/appnexusBidAdapter.js

```diff
@@ -519,6 +570,62 @@ function hasDebug(bid) {
   return !!bid.debug
 }
 
+function hasAdPod(bid) {
+  return (
+    bid.mediaTypes &&
+    bid.mediaTypes.video &&
+    bid.mediaTypes.video.context === 'adpod'
```

Comment:
We added adpod context to mediaTypes.js file here. 
https://github.com/prebid/Prebid.js/pull/3523/
You can update to use this const.